addons/skin.estuary/xml/SkinSettings.xml

```diff
@@ -251,6 +251,31 @@
 					<onclick>Skin.ToggleSetting(HomeMenuNoWeatherButton)</onclick>
 				</control>
 			</control>
+			<control type="grouplist" id="622">
+				<top>133</top>
+				<left>0</left>
+				<right>0</right>
+				<bottom>140</bottom>
+				<onleft>9000</onleft>
+				<onright>60</onright>
+				<onup>610</onup>
+				<pagecontrol>60</pagecontrol>
+				<ondown>610</ondown>
+				<visible>Container(9000).HasFocus(4)</visible>
+				<control type="radiobutton" id="623">
+					<label>$LOCALIZE[31172]</label>
+					<include>DefaultSettingButton</include>
+					<selected>Skin.HasSetting(OSDAutoClose)</selected>
+					<onclick>Skin.ToggleSetting(OSDAutoClose)</onclick>
+				</control>
+				<control type="button" id="624">
+					<label>$LOCALIZE[31173]</label>
+					<label2>$VAR[SkinSettingOSDAutoCloseTime]</label2>
+					<include>DefaultSettingButton</include>
+					<visible>Skin.HasSetting(OSDAutoClose)</visible>
```

Comment:
I'm not a fan of hiding settings in this way with a Visibility condition, instead I'd used an enable condition.

There are examples of this in the Main Menu section

![image](https://user-images.githubusercontent.com/5781142/167258923-01302aa5-9f05-48c3-a4ca-bdd1e8a7d404.png)

![image](https://user-images.githubusercontent.com/5781142/167258948-2382f99f-bd3d-42bc-ac84-7507a5e78d9b.png)

So doing this for for the OSD settings gives

![image](https://user-images.githubusercontent.com/5781142/167258986-621e1ddb-e014-4f0a-b3c1-e0ee9d4d71d6.png)

![image](https://user-images.githubusercontent.com/5781142/167258997-d6e93ccd-4ff3-4123-81dc-04a3c437c209.png)

Code is L271 to L277 here https://gist.github.com/jjd-uk/9f360100d584cb4af3fe55318b085d38